benchmark_utils/__init__.py

```diff
@@ -3,4 +3,156 @@
 # name `benchmark_utils`, and code defined inside will be importable using
 # the usual import syntax
 
-# on peut définir ici des fonctions 
+from benchopt import safe_import_context
+
+
+# Protect the import with `safe_import_context()`. This allows:
+# - skipping import to speed up autocompletion in CLI.
+# - getting requirements info when all dependencies are not installed.
+with safe_import_context() as import_ctx:
+    import numpy as np
+    from braindecode.preprocessing import (
+        preprocess,
+        Preprocessor,
+    )
+    from braindecode.preprocessing import create_windows_from_events
+    from braindecode.augmentation import ChannelsDropout, SmoothTimeMask
+    import torch
+
+
+def list_train_test(test, list):
+    list_test = [list[test]]
+    list_train = list[:test] + list[test + 1:]
+    return list_test, list_train
+
+
+def transformX_moabb(X):
+    """Transform X from moabb to numpy array.
+    Parameters:
+    -----------
+    X: ndarray
+        Data to transform.
+    Returns:
+    --------
+    X0: ndarray
+        Transformed data.
+    """
+    X0 = []
+    for i in range(len(X)):
+        X0.append(X[i])
+    X0 = np.array(X0)
+    return X0
+
+
+def windows_data(dataset, paradigm_name):
+    """Create windows from the dataset.
+
+    Parameters:
+    -----------
+    dataset: MOABBDataset
+        Dataset to use.
+    paradigm_name: str
+        Name of the paradigm to use.
+    Returns:
+    --------
+    windows_dataset: WindowsDataset
+        Dataset with windows.
+    sfreq: float
+        Sampling frequency of the dataset.
+    """
+
+    # Define mapping of classes to integers
+    # We use two classes from the dataset
+    # 1. left-hand vs right-hand motor imagery
+    if paradigm_name == "LeftRightImagery":
+        mapping = {"left_hand": 1, "right_hand": 2}
+
+    elif paradigm_name == "MotorImagery":
+        mapping = {"left_hand": 1, "right_hand": 2, "feet": 4, "tongue": 3}
```

Comment:
This seems very hard coded, and it will break if we extend to other paradigm.
Can't you get this by passing the paradigm as an argument of the function and creating the mapping on the fly?